administration/src/cards/CsvFactory.ts

```diff
@@ -0,0 +1,67 @@
+import { QrCode } from '../generated/card_pb'
+import { convertProtobufToHexCode } from '../util/qrcode'
+import CardBlueprint from './CardBlueprint'
+import { CreateCardsResult } from './createCards'
+import AddressExtensions from './extensions/AddressFieldExtensions'
+import BirthdayExtension from './extensions/BirthdayExtension'
+import NuernbergPassIdExtension from './extensions/NuernbergPassIdExtension'
+import StartDayExtension from './extensions/StartDayExtension'
+import { findExtension } from './extensions/extensions'
+
+export class CsvError extends Error {
```

Comment:
hm if i throw somewhere an csv error it should be handled it the `useCardGenerator` i think and show a proper toast. Currently if i click on csv export and an error occurs here i just get
`Etwas ist schiefgegangen beim erstellen der PDF.` even i didn't create the a pdf

Reply:
Good catch. Adjusted the error handling, should work now.